cli/src/commands/install.js

```diff
@@ -265,7 +266,17 @@ async function installNpmLibDefs({
       const termMatches = term.match(/(@[^@\/]+\/)?([^@]+)@(.+)/);
       if (termMatches == null) {
         const pkgJsonData = await getPackageJsonData(cwd);
-        const pkgJsonDeps = getPackageJsonDependencies(pkgJsonData, []);
+        const workspacesPckJsonData = await findWorkspacesPackages(
```

Comment:
Nitpick but suggest we rename this var to be consistent with how it's used in rest of the project
```suggestion
        const workspacesPkgJsonData = await findWorkspacesPackages(
```